pkg/services/ngalert/state/historian/loki.go

```diff
@@ -304,6 +304,13 @@ func StatesToStream(rule history_model.RuleMeta, states []state.StateTransition,
 			state.State.Values = map[string]float64{}
 		}
 
+		thresholdInputValue := 0.0
+		if state.State.State == eval.Alerting {
```

Comment:
why only alerting?

Reply:
I thought it was in the ticket..
My mistake fixing